src/python/grpcio/grpc/_cython/_cygrpc/aio/channel.pyx.pxi

```diff
@@ -18,13 +18,13 @@ cdef class AioChannel:
         self._target = target
 
     def __repr__(self):
-        class_name = self.__class__.__name__ 
+        class_name = self.__class__.__name__
         id_ = id(self)
         return f"<{class_name} {id_}>"
 
     def close(self):
         grpc_channel_destroy(self.channel)
 
-    async def unary_unary(self, method, request):
+    async def unary_unary(self, method, request, timeout=None):
```

Comment:
Our pattern in the past has been to use keyword arguments on our public API surface for breathing room for API additions, but for internal APIs such as this, we tend to only use positional arguments, since we're completely in control of the call sites as well as the function signature.